sn_networking/src/lib.rs

```diff
@@ -78,14 +80,16 @@ pub const fn close_group_majority() -> usize {
     CLOSE_GROUP_SIZE / 2 + 1
 }
 
+/// Max duration for all GET attempts
+const MAX_GET_RETRY_DURATION_MS: u64 = 6800;
+const MAX_GET_RETRY_DURATION: Duration = Duration::from_millis(MAX_GET_RETRY_DURATION_MS);
+/// Max duration for all PUT attempts
+const MAX_PUT_RETRY_DURATION: Duration = Duration::from_millis(MAX_GET_RETRY_DURATION_MS * 3);
+
 /// Max duration to wait for verification
-const MAX_REVERIFICATION_WAIT_TIME_S: std::time::Duration = std::time::Duration::from_millis(750);
+const MAX_REVERIFICATION_WAIT_TIME_MS: Duration = Duration::from_millis(750);
 /// Min duration to wait for verification
-const MIN_REVERIFICATION_WAIT_TIME_S: std::time::Duration = std::time::Duration::from_millis(300);
-/// Number of attempts to GET a record
-const GET_RETRY_ATTEMPTS: usize = 3;
-/// Number of attempts to PUT a record
-const PUT_RETRY_ATTEMPTS: usize = 10;
+const MIN_REVERIFICATION_WAIT_TIME_MS: Duration = Duration::from_millis(300);
```

Comment:
do we have an estimation of steps of those delays to be generated by the backoff ?
300ms or 750ms feels too short for a public testnet to complete a  network option.
i.e. after the firmt operation, the `max_elapsed_time` may got reached already, hence there will be no further attempt in case of error?

Reply:
https://docs.rs/backoff/latest/backoff/

it retries exponentially over 10s by default. 

we also have this min_reverification wait time _before_ we start doing verification reqs.